tests/controls/bitmapcomboboxtest.cpp

```diff
@@ -82,9 +79,13 @@ void BitmapComboBoxTestCase::Bitmap()
     wxArrayString items;
     items.push_back("item 0");
     items.push_back("item 1");
-
+#if defined __WXMSW__ || defined __WXGTK__
     //We need this otherwise MSVC complains as it cannot find a suitable append
     static_cast<wxComboBox*>(m_combo)->Append(items);
+#else
+    for( unsigned int i = 0; i < items.size(); ++i )
+        m_combo->Append(items[i]);
+#endif
```

Comment:
It would be better to just use the same code for all platforms:
```suggestion
    for( unsigned int i = 0; i < items.size(); ++i )
        m_combo->Append(items[i]);
```

Perhaps we could add this `Append()` overload to `wxBitmapComboBoxBase` to make it available in the generic version too, but for now the above should be good enough and avoid the ugly cast.